src/docs/pages/introduction/getting-started/vue.md

```diff
@@ -0,0 +1,109 @@
+---
+eleventyNavigation:
+  key: Vue
+  parent: Getting started
+  order: 2
+layout: layout.njk
+title: Vue
+permalink: introduction/getting-started/vue/
+---
+
+# Vue
+
+Liquid Oxygen comes with [Vue](https://vuejs.org/) bindings for all Web Components. Although the rendered components are still Web Components, the bindings improve the developer experience by providing a more familiar API and a better integration with Vue.
+
+On this page, you'll find detailed instructions on how to integrate Liquid Oxygen into your Vue project and how to use the components.
+
+## Prerequisites
+
+Liquid Oxygen is easy to integrate into an existing Vue project. We assume you already have a project set up. If not, you can create a new project with [Vite](https://vitejs.dev/).
+
+```sh
+npm create vite@latest your-project-name -- --template vue-ts
+```
+
+For more information about this command and Vite, please refer to the [Vite documentation](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).<br />Although this guide assumes your project is using typescript, all examples should also be applicable to javascript projects.
+
+## Install
+
+Add Liquid Oxygen to your project with the package manager of your choice.
+
+```sh
+npm install @emdgroup-liquid/liquid
+```
+
+<ld-notice>
+  Liquid Oxygen works with any package manager (e.g. <code>npm</code>, <code>yarn</code> or <code>pnpm</code>). For simplicity, we use <code>npm</code> in this guide.
+</ld-notice>
+
+### Import stylesheet
+
+All Web Components are loaded togehter with their styles embedded. Therefore we only need to import the global stylesheet for Liquid Oxygen.
+
+Add the following code to your `App.vue` file (or any similar file which is loaded for every page).
+
+```html
+// App.vue
+<script setup lang="ts">
+  import '@emdgroup-liquid/liquid/dist/css/liquid.global.css'
+</script>
+```
+
+## Usage
+
+When adding Liquid Oxygen components to a Vue project, it is crucial to use the Vue bindings. All components are imported from `@emdgroup-liquid/liquid/dist/vue`. The bindings significantly improve Vue compatibility and your developer experience.
+
+Let's have a look at how to add a [LdButton](components/ld-button/) to your project. This examnple also includes a [LdIcon](components/ld-icon/) as it helps you to check if Liquid Oxygen assets are loaded correctly.
+
+```html
+// SampleComponent.vue
+<script lang="ts">
+import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/vue'
+<script>
+
+<template>
+  <LdButton>
```

Comment:
We should use complete examples for Vue here, including the component definition, i.e.:

```html
<script lang="ts">
import { LdTypo } from '@emdgroup-liquid/liquid/dist/vue'
import { defineComponent } from 'vue'

export default defineComponent({
  name: 'App',
  components: {
    LdTypo,
  },
})
</script>

<template>
  <ld-typo>Hello</ld-typo>
</template>
```

Please take note of the `kebab-case` component tags within the template.

Reply:
Based on the stencil documentation defineComponent is not required for lazy dist components which we still unfortunately use because of an issue with tree-shaking of the react / vue output target. https://github.com/ionic-team/stencil-ds-output-targets/issues/255 

Docs for vue & stencil: https://stenciljs.com/docs/vue

although I will adjust all exmples using kebab-case. 
Additionally I add another script tag including setup as this seems to be necessary when importing the components.